src/nodes/accessors/Camera.js

```diff
@@ -37,7 +39,39 @@ export const cameraProjectionMatrixInverse = /*@__PURE__*/ uniform( 'mat4' ).lab
  *
  * @type {UniformNode<mat4>}
  */
-export const cameraViewMatrix = /*@__PURE__*/ uniform( 'mat4' ).label( 'cameraViewMatrix' ).setGroup( renderGroup ).onRenderUpdate( ( { camera } ) => camera.matrixWorldInverse );
+export const cameraViewMatrix = /*@__PURE__*/ ( Fn( ( { camera } ) => {
```

Comment:
Do you mind changing `cameraProjectionMatrix` in the same fashion? Well will need this for `XRManager`.